mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -676,13 +697,16 @@
         <Alignment value="Negative"/>
         <InfoIcon value="Assets\Textures\btn-upgrade-terran-interferencematrix.dds"/>
         <EditorCategories value="AbilityorEffectType:Units"/>
-        <Duration value="15"/>
+        <Duration value="11"/>
         <ExpireEffect value="ImmortalBarrierCDFix"/>
         <Modification>
             <ModifyFlags index="DisableAbils" value="1"/>
+            <ModifyFlags index="EnableAttack" value="1"/>
             <ModifyFlags index="EnableMove" value="1"/>
             <StateFlags index="SuppressAttack" value="1"/>
             <StateFlags index="SuppressCloak" value="1"/>
+            <StateFlags index="SuppressCombat" value="1"/>
+            <AbilClassEnableArray index="CAbilStop" value="1"/>
```

Comment:
Fixed an issue where units affected by Interference Matrix could not be issued Stop commands